nix/lib/ci.nix

```diff
@@ -1,8 +1,5 @@
-let
-  # Generic nixpkgs, use *only* for lib functions that are stable across versions
-  pkgs = import (import ../sources.nix).nixpkgs { };
```

Comment:
We need this because the `ci.nix` funcitons (especially `dimension`) need to be usable *before* we import the main `nixpkgs`, since we're going to use `dimension` to control *how* we import that nixpkgs. Not clear from the comment, admittedly.